src/components/OfferBox/OfferBox.js

```diff
@@ -2,14 +2,18 @@ import React from 'react';
 import './OfferBox.scss';
 import ProductImage from 'components/ProductImage/ProductImage';
 import ObserveBtn from 'components/ObserveBtn/ObserveBtn';
+import {Link} from 'react-router-dom';
+
 
 const OfferBox = ({image, title, price, place, label, saved, id}) => {
     return (
         <li className='offer-wrapper'>
             <ProductImage image={image} label={label}/>
             <div className='offer-box'>
                 <div className='title-price'>
-                    <p className='title'>{title}</p>
+                    <p className='title'>
+                        <Link to={`/${id}`}>{title}</Link>
+                    </p>
```

Comment:
To może być sam link z klasą, nie potrzebujesz tutaj <p>. Pamiętaj, że link jest elementem liniowym, a paragraf blokowym